bls/src/main/java/tech/pegasys/teku/bls/hashToG2/HashToCurve.java

```diff
@@ -57,28 +65,27 @@ public static boolean isInGroupG2(ECP2 point) {
    * Hashes to the G2 curve as described in the new BLS standard.
    *
    * @param message the message to be hashed. This is usually the 32 byte message digest
-   * @param cipherSuite the salt value for HKDF_Extract
+   * @param dst the domain separation tag (DST)
    * @return a point from the G2 group representing the message hash
    */
-  public static ECP2 hashToG2(Bytes message, Bytes cipherSuite) {
+  public static ECP2 hashToG2(Bytes message, Bytes dst) {
```

Comment:
Is there a specification for this method anywhere? It would be really useful for reviewing if there was.

Reply:
Hmm - it's complicated as it pulls together bits from across the hash-to-curve standard. But you're right, it would be useful to be explicit what's happening here. I'll try to add the relevant refs.